planning/autoware_boundary_departure_checker/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.14)
```

Comment:
Should this be in the planning?
I feel that as the lane_departure_checker was in the control component, other components than planning may refer to this class. Therefore, moving this package to the common components sounds better to me.

Reply:
@takayuki5168 
lane departure checker will be removed once boundary departure prevention module is done. So it make sense to place it here.

Reply:
@zulfaqar-azmi-t4 
I don't refer to the current node structure. There may be enough possibilities that the control side will check the boundary departure, and in that case, we don't want to move the package's place so often.
The boundary departure feature is not obviously only the planning related feature, and as we have motion_utils, trajectory packages in the common directory, I still think this package should be in the common directory.

Reply:
I see. In that case, I'll move this package to common directory. I will put this PR into draft for now. Once I moved them, I'll reopen the PR.

Reply:
Thank you so much.